superduperdb/base/config.py

```diff
@@ -50,18 +50,27 @@ class Retry(BaseConfigJSONable):
 class Cluster(BaseConfigJSONable):
     """Describes a connection to distributed work via Dask
 
-    :param dask_scheduler: The Dask scheduler URI
     :param backfill_batch_size: The number of rows to backfill at a time
                                 for vector-search loading
+    :param compute: The URI for compute i.e 'local', 'dask+tcp://localhost:8786'
+                    None: Run all jobs in local mode i.e simple function call
```

Comment:
The ':' at the end is confusing -- is that part of the string or part of the explanation. You need to add "quotation" marks.